gear-programs/erc20-relay/app/tests/gclient.rs

```diff
@@ -38,12 +38,6 @@ async fn gas_for_reply() {
         .unwrap();
 
     let mut client = Erc20Relay::new(remoting.clone());
-    while client
```

Comment:
Since we are changing the program logic completely this test case is not actual anymore